scripts/airgapped/get-all-images.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/bash
+#
+# This script parses given bundle file for github repositories and branches. Then checks out each
+# charm's repository one by one using specified branch and collects images referred by that charm
+# using that repository's image collection script
+#
+BUNDLE_FILE=$1
+IMAGES=()
+# retrieve all repositories and branches for CKF
+REPOS_BRANCHES=($(yq -r '.applications[] | to_json' $BUNDLE_FILE | jq -r 'select(._github_repo_name) | "\(._github_repo_name):\(._github_repo_branch)"' | sort --unique))
+RESOURCE_DISPATCHER_BRANCH=track/1.0
```

Comment:
Yes, it's because the resource dispatcher is not in the bundle and we want to test MLFlow with.

Initially when we were working on the airgap bundle we had tried to define an mlops bundle that includes everything https://github.com/canonical/bundle-kubeflow/pull/712. Personally I think we should go towards that direction, which will simplify testing and also scripts like these.

But this will need some discussion with Andreea as well, since it's a bit of product definition. For now I'll 
1. create an issue that mentions that those are hardcoded and we need to move away from this
2. link the issue with a TODO

Reply:
Created an issue and added the todo in the code
https://github.com/canonical/bundle-kubeflow/issues/789
https://github.com/canonical/bundle-kubeflow/issues/787

https://github.com/canonical/bundle-kubeflow/pull/682/commits/3a73ee154ed3a67b4b275faf47adf7e976961148